offchain/rollups-events/src/broker/mod.rs

```diff
@@ -346,7 +346,7 @@ pub enum BrokerError {
     InvalidPayload { source: serde_json::Error },
 }
 
-#[derive(Debug, Parser)]
+#[derive(Debug, Parser, Clone)]
```

Comment:
I'd consider whether the cases you're cloning the config could instead just borrow.

Reply:
The borrow requires a lifetime specifier, the flatten command also complains, and the other configs from block-history and state-fold are cloned, so I just kept the pattern.